### ISSUE TITLE
7.7.6 SuiteP very limited number of favorites shown

### ISSUE THREAD

**Issue by bmwtourer** [salesagility/SuiteCRM#2541]
favorites is a popular feature for many users.
One of my users who was using SuiteP  asked me why she does not see all of them (she has many).
When switching to Suite R we see all of them.

I do not have the exact count but SuiteP would show less than 10 favorites while SuiteR showed more than 20...
it was reported using Chrome but I would be surprise that its linked to the browser of the server OS.
running from Windows server 2008 IIs and MSSQL 2012

**Comment by shogunpol** [comment]
@bmwtourer, thanks very much, we will consider your suggestion. To clarify, you are talking about favourites on right navbar under recently viewed?

**Comment by bmwtourer** [comment]
.... navbar  ... located on the **left**!   :-)
yes under recently viewed...

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [7.7.6 SuiteP very limited number of favorites shown](https://trello.com/c/wzxGwRUv/58-7-7-6-suitep-very-limited-number-of-favorites-shown)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
